cmd/sdn-cni-plugin/openshift-sdn_linux.go

```diff
@@ -234,7 +236,30 @@ func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	return result.Print()
+	result.Interfaces = nil
+	result.IPs[0].Interface = nil
+	result.IPs[0].Gateway = defaultGW
```

Comment:
I wouldn't do this; just leave the Interfaces and IPs array alone and correct for a 0.3.x Result type. If the runtime requested a different version (eg 0.1 or 0.2) then convertToRequestedVersion() will handle that downconvert.

Reply:
Done.